### DISCUSSION TITLE
Deactivating virtual environment

### DISCUSSION THREAD

**Discussion by GNamithaReddy** [Atri-Labs/atrilabs-engine#600] [Help: Installation & Start] [answered]
How to stop current atri process?

**Comment by darshitac11** [comment, marked as answer]
@GNamithaReddy - To stop Atri editor, you can terminate the process in the terminal using `Ctrl + C` (the same key combination that terminates any process). 

To deactivate your virtual environment, you should use `exit` command if you are using `pipenv` or `conda deactivate` if you are using conda. 

Learn more about the development flow for an existing project here: https://docs.atrilabs.com/existing-project/dev-flow-existing-proj#deactivate-virtual-environment-at-the-end-of-a-development-session
